src/elm/Page/Settings.elm

```diff
@@ -0,0 +1,528 @@
+            LoggedIn.mutation loggedIn
+                (Cambiatus.Mutation.preference
+                    (\optionalArgs ->
+                        case notification of
+                            Transfer ->
+                                { optionalArgs | transferNotification = OptionalArgument.Present newValue }
+
+                            Claim ->
+                                { optionalArgs | claimNotification = OptionalArgument.Present newValue }
+
+                            Digest ->
+                                { optionalArgs | digest = OptionalArgument.Present newValue }
+                    )
+                    preferencesSelectionSet
+                )
+                (CompletedTogglingNotification notification)
+
+        updateModel model =
+            case notification of
+                Transfer ->
+                    { model | transferNotificationStatus = UpdatingTo newValue }
+
+                Claim ->
+                    { model | claimNotificationStatus = UpdatingTo newValue }
+
+                Digest ->
+                    { model | digestNotificationStatus = UpdatingTo newValue }
+    in
+    UR.mapModel updateModel
+        >> UR.addExt updatePreferencesMutation
+
+
+
+-- VIEW
+
+
+view : LoggedIn.Model -> Model -> { title : String, content : Html Msg }
+view loggedIn model =
+    { title = "TODO"
+    , content =
+        div [ class "container mx-auto px-4 mt-6 mb-20" ]
+            ([ viewAccountSettings
+             , viewNotificationPreferences loggedIn model
+             , [ viewPdfErrorModal loggedIn model ]
+             ]
+                |> List.concat
+            )
+    }
+
+
+viewPdfErrorModal : LoggedIn.Model -> Model -> Html Msg
+viewPdfErrorModal loggedIn model =
+    View.Modal.initWith
+        { closeMsg = ClosedPdfErrorModal
+        , isVisible =
+            case model.pdfDownloadStatus of
+                PdfOk ->
+                    False
+
+                EncryptedPassphraseNotPresent ->
+                    True
+        }
+        -- TODO - I18N
+        |> View.Modal.withHeader "Sorry, we can't find your 12 words"
+        |> View.Modal.withBody
+            [ p [ class "my-3" ]
+                -- TODO - I18N
+                [ text "Please, check if you have your 12 words saved during the registration process and use them for further signing in." ]
+            , p [ class "my-3" ]
+                -- TODO - I18N
+                [ text "If you completely lost your 12 words, please, contact us and provide this private key and we will help you to recover:"
+                ]
+            , case LoggedIn.maybePrivateKey loggedIn of
+                Nothing ->
+                    text ""
+
+                Just pk ->
+                    p [ class "font-bold my-3 text-lg text-center border p-4 rounded-sm bg-gray-100" ]
+                        [ text (Eos.Account.privateKeyToString pk)
+                        ]
+            ]
+        |> View.Modal.toHtml
```

Comment:
@lucca65 do you think we should still worry about this? Here's the reasoning behind it, from `index.js` (the linked PR was merged in July 2020):

```js
const store = JSON.parse(getItem(USER_KEY))
const pin = arg.data.pin

// `.encryptedPassphrase` property was added in https://github.com/cambiatus/frontend/pull/270 while redesigning
// the Profile page. For the users who were already logged-in before these changes were introduced,
// this property may not exist. This case is handled by passing `isDownloaded: false` to Elm
// for further processing.
if (store.encryptedPassphrase) {
  const decryptedPassphrase = sjcl.decrypt(pin, store.encryptedPassphrase)
  return downloadPdf(store.accountName, decryptedPassphrase)
} else {
  // The case when there's not passphrase stored in user's browser, only the Private Key
  return { isDownloaded: false }
}
```

Reply:
I don't bro, this was a real concern back then, as we were still adjusting the wallet, such a core part of the app. Its been stable for a while and I don't think we need to worry about sessions that old. It should actually fail for those users. 

The main concern was that users might loose access to their account completely as back then we supported 12 words or private key. Depending on the method they could be locked out their own account. Since then we dropped PK support.